changelogs/unreleased/gh-8100-parser-changes.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/sql
+
+* Unsupported sort order is dropped from index creation statements (gh-8100).
```

Comment:
Looks like a breaking change. Please check with @Totktonada if we should mark it as breaking in the changelog or add a compat option for it.

Reply:
Decided to leave as it is now.